FluentTc/Domain/Build.cs

```diff
@@ -14,6 +14,7 @@ public interface IBuild
         DateTime QueuedDate { get; }
         BuildConfiguration BuildConfiguration { get; }
         Agent Agent { get; }
+        TestOccurrences TestOccurrences { get; }
```

Comment:
I know that this is confusing, but FluentTc is under a refactoring process. During the process concrete classes are replaced with read-only interfaces. It means interfaces with only getters. Build class was already refactored to IBuild interface and it has only getters. This approach makes testing much easier. Can you please replace TestOccurences class to an interface?